server/src/controllers/feedback.py

```diff
@@ -16,11 +15,15 @@ def compute_feedback(request: dict):
     request["feedbackEngine"]["tenantId"] = settings.tenant_id
     # Nasty hack as the analytics engine only returns masteries for this user
     # Note: the bearer token used to request from the analytics engine needs to belong to this user
-    request["feedbackEngine"]["userId"] = os.environ.get('VOCABTRAINER_USER')
+    request["feedbackEngine"]["userId"] = settings.vocabtrainer_user
 
     response = httpx.post(url=url, headers=headers, json=request)
 
-    return response.json()
+    json_response = response.json()
```

Comment:
maybe try catch this as the method can throw

Reply:
if it throws, can we recover from it? is there a benefit from not just letting it throw?

Reply:
> can we recover from it?
Probably not, at least I don't think e.g. a retry would help. 

> is there a benefit from not just letting it throw?
We could forward to the front-end a more meaningful error response. Or at least make sure the error json is structurally consistent with other responses the backend returns. Not sure if fastapi takes already care of this when this method throws  its exception.

Reply:
>  it seems that's not done anywhere in the app yet.

Yeah, just some professional `print(e)` statements were used :see_no_evil: Feel free to adjust it elsewhere as well since you now introduced the proper logger :+1: